src/test/java/com/google/sps/CommentServiceTest.java

```diff
@@ -0,0 +1,90 @@
+package com.google.sps;
+
+import com.google.sps.servlets.CommentService;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+
+public final class CommentServiceTest {
+
+  CommentService commentService = new CommentService();
+
+  /**
+   * A list of Strings should be returned for any public video that has comments. 
+   */
+  @Test
+  public void publicVideoReturnsCommentsList() throws GeneralSecurityException, IOException {
+    String videoId = "v3whnmU619I"; // Public video that has some comments.
```

Comment:
General comment for all unit tests. You shouldn't be calling other services in unit test. Unit test should be self contained. You should mock the service calls to return hard coded responses. In practice, unit tests are run many times automatically making sure things are still working. You wouldn't want to waste your quota on unit tests.